### ISSUE TITLE
question about consistency mechanism

### ISSUE THREAD

**Issue by junghan0611** [BenTsai7/SCM-DRAM-FPTree#1]
FPTree uses a micro-log for data consistency.
I read the code for a little. I could not find any codes related to micro-log. 
I don't know why you didn't implement the micro-log. Any idea?  
Thanks...

**Comment by BenTsai7** [comment]
Yes. It's just a simple FPTree implemention. We implement the low-level NVM-DRAM based leaf allocator , fingerprint hashing, and basic operation on Bplus tree,   However, micro-log and HTM-lock are not implemented. The codes are just for reference only.
